src/scss/includes/search_form.scss

```diff
@@ -254,6 +256,12 @@ input[type="search"] {
   padding: 9px 18px;
 }
 
+.autocomplete_error {
+  padding: 15px 10px 5px;
+  font-size: 16px;
+  color: $primary_text;
+}
```

Comment:
A `cursor: default` rule is probably required like for the separator, so the user doesn't think this is an active element.